content/ia/0004-satisfacao-restricoes.md

```diff
@@ -478,8 +477,7 @@ substancialmente mais eficientes.
 Encontram-se abaixo dois exemplos da mesma procura, que seguem ordenações das variáveis
 por nível diferentes, por forma a ilustrar as diferenças que pequenas alterações podem surtir.
 Note-se que, para o mesmo problema (problema este relativamente simples, com poucas variáveis
-e restrições), conseguimos um ganho de $50\%$ em desempenho: conseguimos cortar metade dos testes de
-consistência realizados!
+e restrições), conseguimos reduzir em $50\%$ o número de testes de consistência realizados!
 
 ![Escolher a ordem das variáveis é relevante](imgs/0004-choosing-variables-matters.svg#dark=2)
```

Comment:
Think `0004-choosing-variables-matters.svg` might have some issues 😅
- The tree leaf with the consistent solution is poorly marked, it's the one next to it (B=6 > C=-1 > A=2)
- There is an arrow out of place connecting two leaves (B=4 > C=-1 > A=1)
- There is a red node that should be blue painted (B=4 > C=4)

Reply:
The image has indeed some mistakes, @D-Card told me so this afternoon as well, I'll fix it asap

Reply:
Fixed in #767